### PR TITLE
Added check for power plan  and numa optimize

### PR DESCRIPTION
**Pull request by asofron**

### CODE REVIEW

File: WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1 [hunk 1]
```diff
@@ -2,58 +2,73 @@
...
-    [String] $hvServer1,
-    [parameter(Mandatory=$true)]
-    [String] $hvServer2,
     [parameter(Mandatory=$false)]
-    [String] $Adapter = "SLOT*",
-    [parameter(Mandatory=$false)]
-    [String] $VMname = "dummy"
+    [String] $hvServer,
+	[parameter(Mandatory=$false)]
+    [String] $VMname,
```
Comment (chvalean) [inline WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1]: these 2 params are not required and not used in the script, please remove them.

File: WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1 [hunk 2]
```diff
@@ -2,58 +2,73 @@
...
+$params = $testParams.Split(";")
+foreach ($p in $params)
+{
+    $fields = $p.Split("=")
+    switch ($fields[0].Trim())
+    {
+    "hvServer1"	   { $hvServer1    = $fields[1].Trim() }
+    "hvServer2"	   { $hvServer2    = $fields[1].Trim() }
+    "DependencyVM" { $DependencyVM = $fields[1].Trim() }
+	"Adapter"      { $Adapter      = $fields[1].Trim() }
```
Comment (chvalean) [inline WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1]: wrong indentation

File: WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1 [hunk 3]
```diff
@@ -2,58 +2,73 @@
...
+    Write-Host "$DependencyVM VM from $hvServer2 runs on NUMA NODE: $numa; keeping VM up to ensure test VM starts on numa node #1"
 }
 else {
-    Stop-VM -Name $VMname -ComputerName $hvServer2 -TurnOff
+    Stop-VM -Name $DependencyVM -ComputerName $hvServer2 -TurnOff
 }
 
 Get-NetAdapter -Name "*SRIOV*"  | Disable-NetAdapter -Confirm:$false
 Get-NetAdapter -Name "*SRIOV*"  | Enable-NetAdapter  -Confirm:$false
+Write-Host "NUMA configure  successfull"
```
Comment (chvalean) [inline WS2012R2/lisa/Infrastructure/Configure-Numa-Rss.ps1]: 2 blank spaces between words, and maybe change to configured*

**Comment by asofron** [comment]
Verified Execution :
running setup script 'Infrastructure\Configure-Numa-Rss.ps1'
dummy VM from LIS-PERF08 runs on NUMA NODE: 1; stopping dependency VM
dummy VM from LIS-PERF09 runs on NUMA NODE: 0; keeping VM up to ensure test VM starts on NUMA node #1
NUMA configured successfully on both servers

**Pull request merged**